### DISCUSSION TITLE
Fallas al instalar Vue/CLI

### DISCUSSION THREAD

**Discussion by ErickJHernandezM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#451] [Q&A]
Buenas a todos, alguien podría ayudarme con esto, es que al instalar el Vue/CLI, me salen errores, ya entre a los links que me da, y ya intente instalar lo que dice ahí, pero sigue igual, incluso volví a instalar Node, ya lo eh intentado varias veces y sigue apareciendo lo mismo. Llevo dos días y no puedo solucionarlo
![vue](https://user-images.githubusercontent.com/99061757/159076307-e098db92-2efe-4ba0-9134-9515f4282a56.png)

**Comment by AngelCruzO** [comment]
Los amarillos no son errores, son warnings. Puedes instalar sin problema y uilizarlo, a mi tambien me los mostro

**Comment by doguedogue** [comment]
En Windows usa tu terminal CMD Administrador
En Linux usa sudo para ejecutar el comando de instalación:
 
`npm install -g @vue/cli`

NOTA: esto se hace para instalaciones globales  (-g) normalmente

**Comment by LuisSantiago10** [comment]
puedes instalar con el puro node sin cli de vue 
1.-npm init vue@latest
2.- cd <proyect>
3.- npm install
4.- npm run dev
